#! /bin/sh /usr/share/dpatch/dpatch-run
## 22-fix-casting.dpatch by  <tsmithe@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix a casting error

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' musescore-1.1~/mscore/mscore/exportxml.cpp musescore-1.1/mscore/mscore/exportxml.cpp
--- musescore-1.1~/mscore/mscore/exportxml.cpp	2011-07-27 14:32:43.000000000 +0100
+++ musescore-1.1/mscore/mscore/exportxml.cpp	2011-08-12 17:00:27.808351229 +0100
@@ -2363,7 +2363,7 @@
                    el->userOff().y());
 */
             if (el->type() == HAIRPIN || el->type() == OTTAVA || el->type() == TEXTLINE) {
-                  SLine* sl = static_cast<const SLine*>(el);
+                  SLine* sl = static_cast<SLine*>(el);
 //                  printf("slin segsz=%d", sl->lineSegments().size());
                   if (sl->lineSegments().size() > 0) {
                         LineSegment* seg = sl->lineSegments().at(0);

